extensions/consul-config/runtime/pom.xml

```diff
@@ -19,8 +19,8 @@
             <artifactId>quarkus-arc</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-jackson</artifactId>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-web-client</artifactId>
```

Comment:
Any reason to not use the Mutiny variant?

Reply:
I wanted to feel the pain of CompleteableFuture :).

In all seriouness, you are right, I will update the PR to use Mutiny later on today.